src/shared_realm.hpp

```diff
@@ -134,6 +135,7 @@ class Realm : public std::enable_shared_from_this<Realm> {
         std::string path;
         // User-supplied encryption key. Must be either empty or 64 bytes.
         std::vector<char> encryption_key;
+        BinaryData realm_data;
 
         bool in_memory = false;
```

Comment:
`in_memory` feels a little ambiguous now that there are two different types of in-memory Realm.